interfaces/builtin/network_control.go

```diff
@@ -286,7 +286,7 @@ capability sys_admin, # for setns()
 network netlink raw,
 
 / r,
-/run/netns/ r,     # only 'r' since snap-confine will create this for us
+/run/netns/ rk,     # no 'w' since snap-confine will create this for us
```

Comment:
For reference, this is in line with how `ip netns` works https://github.com/iproute2/iproute2/blob/f22c49730c3691c25a1147081363eb35aa9d1048/ip/ipnetns.c#L819